sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/ConditionEvaluatesToConstantTest.cs

```diff
@@ -52,9 +55,16 @@ public void ConditionEvaluatesToConstant_FromCSharp7()
         public void ConditionEvaluatesToConstant_FromCSharp8()
         {
             Verifier.VerifyAnalyzer(@"TestCases\ConditionEvaluatesToConstant.CSharp8.cs",
-                new ConditionEvaluatesToConstant(),
+                GetAnalyzer(),
                 ParseOptionsHelper.FromCSharp8,
                 additionalReferences: NuGetMetadataReference.NETStandardV2_1_0);
         }
+
+        private static SonarDiagnosticAnalyzer GetAnalyzer()
+        {
+            // Symbolic execution analyzers are run by the SymbolicExecutionRunner
+            var analyzers = ImmutableArray.Create<ISymbolicExecutionAnalyzer>(new ConditionEvaluatesToConstant());
```

Comment:
nitpick: I think you can make this an arrow method and move the `ImmutableArray.Create` as parameter to the factory

`analyzers` suggests there could be more, and in this UT we don't expect more, so by having an arrow method you'd simplify things